app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Parent exception for all vaccine-related issues."""
+    pass
```

Comment:
According to the checklist, you should use descriptive error messages. Currently, your custom exceptions do not provide any error messages.